Sources/Networking/Operations/PostReceiptDataOperation.swift

```diff
@@ -259,7 +268,7 @@ extension PostReceiptDataOperation.PostData: Encodable {
         try container.encodeIfPresent(self.testReceiptIdentifier, forKey: .testReceiptIdentifier)
     }
 
-    var fetchToken: String { return self.receiptData.asFetchToken }
+    var fetchToken: String { return self.receipt.serialized() }
```

Comment:
Awesome

Reply:
This reminds me, let's add a new test in `BackendPostReceiptDataTests` that posts a JWS receipt.